tests/integration/roleTemplate_test.go

```diff
@@ -118,3 +118,35 @@ func (m *IntegrationSuite) TestRoleTemplateNoAPIGroups() {
 	}
 	validateEndpoints(m.T(), endPoints, m.clientFactory)
 }
+
+func (m *IntegrationSuite) TestRoleTemplateInvalidContext() {
+	newObj := func() *v3.RoleTemplate { return &v3.RoleTemplate{} }
```

Comment:
This test can maybe be named a bit better?

Someone reading it would need to figure out that the only significant thing in this test is the `invalidCreate.ProjectCreatorDefault` line?

Also, maybe we could improve the validation a bit?

Reply:
> maybe we could improve the validation a bit?

can you elaborate how